tester_app/client/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "react-webpack-rails-tutorial",
+  "version": "1.1.0",
+  "description": "Code from the React Webpack tutorial.",
+  "main": "server.js",
+  "engines": {
+    "node": "4.2.0",
+    "npm": "3.3.6"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial.git"
+  },
+  "keywords": [
+    "react",
+    "tutorial",
+    "comment",
+    "example"
+  ],
+  "author": "justin808",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
+  },
+  "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js",
+    "build:client": "NODE_ENV=production webpack --config webpack.client.rails.config.js",
+    "build:server": "NODE_ENV=production webpack --config webpack.server.rails.config.js",
+    "build:dev:client": "webpack -w --config webpack.client.rails.config.js",
+    "build:dev:server": "webpack -w --config webpack.server.rails.config.js",
+    "lint": "npm run eslint && npm run jscs",
+    "eslint": "eslint --ext .js,.jsx .",
+    "jscs": "jscs --verbose ."
+  },
+  "dependencies": {
+    "axios": "^0.7.0",
```

Comment:
:+1:  Plus we need to consider if axios is still our recommendation rather than isomorphic-fetch.